### ISSUE TITLE
performance of your provided P3D Resnet trained on Kinetics dataset

### ISSUE THREAD

**Issue by caidonkey** [ZhaofanQiu/pseudo-3d-residual-networks#27]
Hi, 

I evaluated your provided P3D Resnet models trained on Kinetics-400 dataset. Specifically following clues mentioned in your paper 'Learning Spatio-Temporal Representation with Pseudo-3D Residual Networks', my data layers for rgb and flow nets are as follow:

layer {
  name: "data"
  type: "VideoData"
  top: "data"
  top: "label_local"
  video_data_param {
    source: "kinetics_400_vallist_rgb_25segs_s1_4eval.txt"
    root_folder:""
    batch_size: 10
    new_length: 16
    new_width: 242
    new_height: 182
    sampling_rate: 1
    show_data: false 
    use_image: true	
  }
  transform_param{
    crop_size: 160
    mirror: false
    mean_value: [104, 117, 123]
  }
  include: { phase: TEST }
}

layer {
  name: "data"
  type: "VideoData"
  top: "data"
  top: "label_local"
  video_data_param {
    source: "kinetics_400_vallist_flow_25segs_s1_4eval.txt"
    root_folder:""
    batch_size: 10
    new_length: 16
    new_width: 242
    new_height: 182
    sampling_rate: 1
    show_data: false 
    use_image: true	
    use_flow: true
  }
  transform_param{
    crop_size: 160
    mirror: false
    mean_value: 128
  }
  include: { phase: TEST }
}

I got 59.47%/81.19% and 69.38%/89.06% clip-level and (average pooling) video-level top1/top5 accuracies on kinetics-400 validation set using your RGB model (p3d_resnet_kinetics_iter_190000.caffemodel).
And I got 9.42%/22.21% and 15.95%/33.66% clip-level and video-level top1/top5 accuracies on Flow modality (p3d_resnet_kinetics_flow_iter_284000.caffemodel).

I guess there must be something wrong with my evaluation. Could you please kindly provide the corresponding performances of your released models for reference?

**Comment by fjzpcmj** [comment]
Hi, @caidonkey , do you find what the problem is in your data layer?
